testng-eclipse-plugin/src/main/org/testng/eclipse/refactoring/ConvertToYamlAction.java

```diff
@@ -65,6 +65,10 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
             TestNGPlugin.log(e1);
           } catch (CoreException e) {
             TestNGPlugin.log(e);
+          } catch (ParserConfigurationException e) {
```

Comment:
these catch blocks failed the travis build.
actually, I don't mind if you catch a more generic Exception